features/steps/steps.py

```diff
@@ -211,14 +239,59 @@ def instance_getter(i,representation_id, representation_type, negative=False):
         if condition(i, representation_id, representation_type):
             return i
 
+def strip_split(stmt, strp = ' ', splt = ' '):
```

Comment:
The default values here don't make a lot of sense. If you split on `' '`, there wouldn't be any `' '` left to strip.

Reply:
I think this approach is fine for now, but I think we later better revisit this using a grammar as well:

- It's more descriptive in terms of what values it can accept.
- It's stricter in making sure you don't supply wrong pattern
- You get much better error messages when it doesn't comply

Reply:
#36